applications/asset_tracker_v2/tests/ui_module/src/fake_init.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef FAKE_TEST_H_
+#define FAKE_TEST_H_
```

Comment:
I'm no expert in naming of testing stuff, but wonder if this is a dummy inmplementation and not a fake 🤔
Anyway, this isn't a request for change on the naming, just curious of the difference

Reply:
Yeah, Agree. Dummy is better.

Reply:
I can suggest `ui_module_test.h` .

Reply:
You also do other things like marking setup() as unused function in this file

Reply:
That works.